packages/app/src/components/inspector/SymbolLayoutRow.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  PinBottomIcon,
+  PinLeftIcon,
+  PinRightIcon,
+  PinTopIcon,
+  SpaceEvenlyHorizontallyIcon,
+  SpaceEvenlyVerticallyIcon,
+} from '@radix-ui/react-icons';
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import {
+  Divider,
+  Label,
+  RadioGroup,
+  Spacer,
+  InputField,
+  LabeledElementView,
+} from 'noya-designsystem';
+import { memo, useCallback, useMemo } from 'react';
+import * as InspectorPrimitives from './InspectorPrimitives';
+
+interface Props {
+  groupLayout?: Sketch.FreeformGroupLayout | Sketch.InferredGroupLayout;
+  setGroupLayout: (value: Sketch.InferredLayoutAxis | '') => void;
+  setLayoutAnchor: (value: Sketch.InferredLayoutAnchor) => void;
+  setMinWidth: (value: number) => void;
+}
+
+export default memo(function SymbolInspector({
+  groupLayout,
+  setMinWidth,
+  setGroupLayout,
+  setLayoutAnchor,
+}: Props) {
+  const layoutValue = useMemo(() => {
```

Comment:
I did what you show for the most part. I only left this line like this instead of placing it with the default Horizontal because it didn't allow to have the value as undefined for showing 'none'

```ts
const layoutAxis = inferredLayout?.axis;
```

I really like how it looks nice way nicer